components/sidebar/sidebar_category/sidebar_category_menu/sidebar_category_menu.tsx

```diff
@@ -214,6 +284,22 @@ class SidebarCategoryMenu extends React.PureComponent<Props, State> {
             </React.Fragment>
         );
     }
+    renderModals = () => {
+        let moreDirectChannelsModal;
+        if (this.state.showDirectChannelsModal) {
+            moreDirectChannelsModal = (
+                <MoreDirectChannels
+                    onModalDismissed={this.hideMoreDirectChannelsModal}
+                    isExistingChannel={false}
+                />
+            );
+        }
+        return (
+            <React.Fragment>
+                {moreDirectChannelsModal}
+            </React.Fragment>
+        );
```

Comment:
I think you might be able to skip the fragment and just return `moreDirectChannelsModal` directly here. There's a chance you might need to set the default to `null` instead `undefined`, but I don't know if that would also be needed.